src/rules/__tests__/test-utils.ts

```diff
@@ -1,6 +1,87 @@
 import { createRequire } from 'module';
+import { TSESLint } from '@typescript-eslint/utils';
+import { version as eslintVersion } from 'eslint/package.json';
+import * as semver from 'semver';
 
 const require = createRequire(__filename);
 const eslintRequire = createRequire(require.resolve('eslint'));
 
 export const espreeParser = eslintRequire.resolve('espree');
+
+export const usingFlatConfig = () => semver.major(eslintVersion) >= 9;
```

Comment:
why does this need to be a function? rather than just

```suggestion
export const usingFlatConfig = semver.major(eslintVersion) >= 9;
```

Reply:
oh yeah, good point!